wdl-lint/src/rules/line_width.rs

```diff
@@ -0,0 +1,131 @@
+//! Ensures that lines do not exceed a certain width.
+
+use wdl_ast::AstToken;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The maximum width of a line.
+const MAX_WIDTH: usize = 90;
```

Comment:
I wonder if we should make this just a field on rule and the visitor (e.g. `LineWidthRule::new(80)`) with `LineWidthRule::default` being implemented that uses 90?